### ISSUE TITLE
Create a new task which shows a list of potential actions user can perform on openshift

### ISSUE THREAD

**Issue by lstocchi** [redhat-developer/openshift-vsts#111]
The extension gives users the chance to execute oc commands but it presume users already have knowledge of oc cli. We should create a task which guides users with the most common actions one can perform in openshift (deploy/delete/..)

**Comment by jeffmaury** [comment]
I think that deserves a spec as I'm not a kubectl expert. As those actions directly mapped onto kubectl commands or is there a higher level mapping ?

**Comment by lstocchi** [comment]
Just shown you this pic to make you better understand what i meant yesterday. Currently we only have `oc command` task which allow users to execute an oc command but what if we want to offer to users a more direct and easy way to accomplish some general task? 

Do you want to deploy an application and don't know much about oc cli? Just use this task, select deploy action, give us your source code and the extension will do all the job. Then we can add more actions based on users request/pm suggestions.

It will come a day where we will create a `odo command` task this is why i wanted to keep this "actions" task separate to the `oc command` task. Does it make any sense?

**Comment by jeffmaury** [comment]
Yes but my fear is that if we don't have a direct mapping onto oc commands then we will have to deal with the variations between oc versions.
